pulsar-client-cpp/test-conf/client-ssl.conf

```diff
@@ -0,0 +1,27 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Pulsar Client configuration
+webServiceUrl=https://localhost:9766/
+brokerServiceUrl=pulsar+ssl://localhost:9886/
+useTls=true
+tlsAllowInsecureConnection=true
```

Comment:
Do we have to set this value to true?

Reply:
I think it is necessary for e2e tls test.
In addition, this file is copy from old `client.conf` and I think this should not be changed in this PR.

Reply:
> In addition, this file is copy from old `client.conf` and I think this should not be changed in this PR.

OK, I see.
However, I think it might be unnecessary if an appropriate certificate is used.